laghos_solver.cpp

```diff
@@ -782,10 +732,23 @@ void LagrangianHydroOperator::UpdateQuadratureData(const Vector &S) const
                // direction of maximal compression. This is used to define the
                // relative change of the initial length scale.
                v.GetVectorGradient(*T, sgrad_v);
+
+               double vorticity_coeff = 1.0;
+               if (use_vorticity)
+               {
+                  const double grad_norm = sgrad_v.FNorm();
+                  const double div_v = fabs(sgrad_v.Trace());
+                  vorticity_coeff = (grad_norm > 0.0) ? div_v / grad_norm : 1.0;
```

Comment:
@camierjs could you propagate these changes (LagrangianHydroOperator::UpdateQuadratureData) to the quadrature kernel?

Reply:
`done`

Reply:
Thank you!